README.md

```diff
@@ -1174,6 +1174,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Kimberly Pangilinan](https://github.com/kimberlypangilinan)
   - [Kimberly Rodriguez](https://github.com/kimmiethecoder)
   - [Kirtan Chandak](https://github.com/kirtanchandak)
+  - [Kirtan Patel] (https://github.com/kirtankp)
```

Comment:
Kindly remove space between name and link as follows..
[Kirtan Patel](https://github.com/kirtankp)

Reply:
okay i will remove the space.